### ISSUE TITLE
Open one Github Pull Request per configuration

### ISSUE THREAD

**Issue by olblak** [updatecli/updatecli#156]
**Is your feature request related to a problem? Please describe.**
Open one Github Pull Request per configuration, at the moment we open one pull request per "target" configuration.
This slows down the time needed to validate a change a merge it.
 
**Describe the solution you'd like**
Ideally I would like to generate an updatecli config checksum before we apply the source stage.
So we wouldn't create a different PR per version change as all commits would target the same branch 

**Additional context**
This only impact the Github scm defined [here](https://github.com/updatecli/updatecli/blob/31665fa37c5ac226a65cd454cb145b236074b9d2/pkg/plugins/github/scm.go#L14), the git scm only push to a targeted configuration

**Comment by olblak** [comment]
This feature has been implemented in version [v0.1.0](https://github.com/updatecli/updatecli/releases/tag/v0.1.0)
We can now use the key `pipeplineID` to specify the branch name like 

```
title: My pipeline
pipelineID: myuniqpipepine <== Allow to specify a branch name which will be use to re-group commits targeting a specific PR
sources:
 ...
conditions:
...
targets:
...
```

**Comment by lemeurherve** [comment]
Why not `branchName` instead of `pipelineID`? It would make more sense IMO
